src/featurecounts/config.vsh.yaml

```diff
@@ -0,0 +1,346 @@
+          description: | 
+            Specify format of the provided annotation file. Acceptable formats include 'GTF' (or compatible GFF format) and 'SAF'. 'GTF' by default. 
+          choices: [GTF, GFF, SAF]
+          default: "GTF"
+          required: false
+        - name: --feature_type
+          alternatives: ["-t"]
+          type: string
+          description: | 
+            Specify feature type(s) in a GTF annotation. If multiple types are provided, they should be separated by ',' with no space in between. 'exon' by default. Rows in the annotation with a matched feature will be extracted and used for read mapping.
+          default: "exon"
+          required: false
+        - name: --attribute_type
+          alternatives: ["-g"]
+          type: string
+          description: | 
+            Specify attribute type in GTF annotation. 'gene_id' by default. Meta-features used for read counting will be extracted from annotation using the provided value.
+          default: "gene_id"
+          required: false
+        - name: --extra_attributes
+          type: string
+          description: | 
+            Extract extra attribute types from the provided GTF annotation and include them in the counting output. These attribute types will not be used to group features. If more than one attribute type is provided they should be separated by comma.
+          required: false
+        - name: --chrom_alias
+          alternatives: ["-A"]
+          type: file
+          description: | 
+            Provide a chromosome name alias file to match chr names in annotation with those in the reads. This should be a two-column comma-delimited text file. Its first column should include chr names in the annotation and its second column should include chr names in the reads. Chr names are case sensitive. No column header should be included in the file.
+          required: false
+          example: chrom_alias.csv
+    
+    - name: Level of summarization
+      arguments:
+        - name: --feature_level
+          alternatives: ["-f"]
+          type: boolean_true
+          description: | 
+            Perform read counting at feature level (eg. counting reads for exons rather than genes).
+    
+    - name: Overlap between reads and features
+      arguments:
+        - name: --overlapping
+          alternatives: ["-O"]
+          type: boolean_true
+          description: | 
+            Assign reads to all their overlapping meta-features (or features if '--feature_level' is specified).
+        - name: --min_overlap
+          type: integer
+          description: | 
+            Minimum number of overlapping bases in a read that is required for read assignment. 1 by default. Number of overlapping bases is counted from both reads if paired end. If a negative value is provided, then a gap of up to specified size will be allowed between read and the feature that the read is assigned to.
+          required: false
+          default: 1
```

Comment:
In that case, would it be better to not specify any default values so that the command runs with actual defaults used in the implementation? Because sometimes setting default to 0 might be a little too strict.

Reply:
Edit: I updated my suggestion from `default: 0` to `example: 1`. I'm not sure what drugs I was taking when I sent this suggestion :zany_face:

Reply:
Sure, no problem. I'll make the changes then.